packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/modelparts/metadata/Card.java

```diff
@@ -0,0 +1,54 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.modelparts.metadata;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+import software.uncharted.terarium.hmiserver.annotations.TSOptional;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Data
+@Accessors(chain = true)
+public class Card {
+    @JsonProperty("DESCRIPTION")
```

Comment:
Unless intentional, you'll probably want to use `JsonAlias` here. When re-serialized these will go out as captials, etc, instead of camel case